src/fixml/data/templates/eval_report.md.jinja

```diff
@@ -0,0 +1,58 @@
+{%- if quarto_header -%}
+---
+title: "{{ title }}"
+format:
+  html:
+    code-fold: true
+    code-summary: "Show the code"
+---
+{%- endif -%}
```

Comment:
```suggestion
{% else %}
```

Reply:
Fixed - This can be fixed by replacing line 9 with `{%- endif %}` without the introduction of the else block.

Reply:
@JohnShiuMK fixed as per our discussion in call, please check again. Thanks!